frame/balances/src/lib.rs

```diff
@@ -477,26 +303,44 @@ pub mod pallet {
 		Withdraw { who: T::AccountId, amount: T::Balance },
 		/// Some amount was removed from the account (e.g. for misbehavior).
 		Slashed { who: T::AccountId, amount: T::Balance },
+		/// Some amount was minted into an account.
+		Minted { who: T::AccountId, amount: T::Balance },
+		/// Some amount was burned from an account.
+		Burned { who: T::AccountId, amount: T::Balance },
+		/// Some amount was suspended from an account (it can be restored later).
+		Suspended { who: T::AccountId, amount: T::Balance },
+		/// Some amount was restored into an account.
+		Restored { who: T::AccountId, amount: T::Balance },
+		/// An account was upgraded.
+		Upgraded { who: T::AccountId },
+		/// Total issuance was increased by `amount`, creating a credit to be balanced.
+		Issue { amount: T::Balance },
+		/// Total issuance was decreased by `amount`, creating a debt to be balanced.
+		Rescind { amount: T::Balance },
```

Comment:
Keep past participle convention?
```suggestion
		/// Total issuance was increased by `amount`, creating a credit to be balanced.
		Issued { amount: T::Balance },
		/// Total issuance was decreased by `amount`, creating a debt to be balanced.
		Rescinded { amount: T::Balance },
```